src/amp/lib/image-fit.test.ts

```diff
@@ -1,6 +1,6 @@
 import { bestFitImage, heightEstimate } from './image-fit';
 
-test('chooses smallest image that is still greater than column width', async () => {
```

Comment:
Is this expected?

Reply:
I'm really not sure tbh. That's weird that it would remove some and not others